Created SortCharsByFrequencyController.

xedisu opened the pull request:
Created SortCharsByFrequencyInputPage.html.
Created SortCharsByFrequencyOutputPage.html
Created StringDTO.java

xedisu committed:
Created SortCharsByFrequencyController.
Created SortCharsByFrequencyInputPage.html.
Created SortCharsByFrequencyOutputPage.html
Created StringDTO.java

BogdanTanase commented on src/main/java/com/algorithmtron/controller/SortCharsByFrequencyController.java:
```diff
@@ -0,0 +1,34 @@
+
+        model.addAttribute("stringDTO", stringDTO);
+
+        return "SortCharsByFrequencyInputPage";
```
Does this need to be with capital letter?

xedisu replied:
you're right! it slipped my eyes. will change. :)

BogdanTanase commented on src/main/java/com/algorithmtron/model/StringDTO.java:
```diff
@@ -0,0 +1,14 @@
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class StringDTO {
```
Consider naming this DTO in a more appropriate matter to better reflect where it is used.

xedisu replied:
Okay, i'll try to find a more suggestive name for it!

xedisu committed:
Created SortCharsByFrequencyController.
Created SortCharsByFrequencyInputPage.html.
Created SortCharsByFrequencyOutputPage.html
Created StringDTO.java

xedisu committed:
Created SortCharsByFrequencyController.
Created SortCharsByFrequencyInputPage.html.
Created SortCharsByFrequencyOutputPage.html
Created StringDTO.java
